src/Pages/GetStartedPage.jsx

```diff
@@ -0,0 +1,281 @@
+      <Container
+        id="documents"
+        component="section"
+        className={classes.regularContainerStyle}
+        maxWidth="md"
+      >
+        <Typography className={classes.headingStyle} variant="h2">
+          Step 1: Documents
+        </Typography>
+        <Typography className={classes.volunteerTextStyle} variant="body1">
+          Gathering documents is the first step in the process. This includes
+          any forms or records that pertain to your conviction. Please gather
+          all relevant court forms and get a copy of your Criminal History
+          Record Information (CHRI).{" "}
+        </Typography>
+        <Typography className={classes.headingStyle} variant="h3">
+          Court Forms
+        </Typography>
+        <CourtFormsFactGroup />
+        <Typography className={classes.headingStyle} variant="h3">
+          Criminal History Record Information (CHRI)
+        </Typography>
+        <Typography className={classes.volunteerTextStyle} variant="body1">
+          You might need a copy of your Criminal History Record Information
+          (CHRI) to vacate your record. There are two options:
+        </Typography>
+        <Paper className={classes.paperStyle}>
+          <Grid container>
+            <Grid item xs={12} sm={6}>
+              <img src={humanProfile} alt="" width="100%" />
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <Box display="flex">
+                <Typography className={classes.volunteerTextStyle}>
+                  Get an{" "}
+                  <Link
+                    rel=""
+                    href="https://watch.wsp.wa.gov/WATCH/Home/Notice?ReturnPage=%2FHome%2FIndex"
+                  >
+                    “unofficial” copy through WATCH
+                  </Link>{" "}
+                  for a $11.00 fee. This report is not always accurate so you
+                  should ask the prosecutor.
+                </Typography>
+              </Box>
+            </Grid>
+          </Grid>
+        </Paper>
+        <Paper className={classes.paperStyle}>
+          <Grid container>
+            <Grid item xs={12} sm={8}>
+              <Typography>
+                If the WATCH printout is not good enough, go to your local
+                police or sheriff’s office for an official “record
+                review/challenge” fingerprint card.
+              </Typography>
+              <Typography className={classes.volunteerTextStyle}>
+                Next, write WSP a letter asking them to send an official copy of
+                your complete CHRI. Include the fingerprint card and a $12 money
+                order. A copy of your CHRI will be returned to the address on
+                the fingerprint card. Send everything to:
+              </Typography>
+            </Grid>
+            <Grid item xs={12} sm={4}>
+              <Typography className={classes.volunteerTextStyle} align="right">
+                <Link align="center">
```

Comment:
nothing happens when I click the address. Or should it not be underlined/linked?